web/landing/partners.js

```diff
@@ -181,12 +182,13 @@ export function Partners() {
                 </LoadingButton>
               )}
               <div>
-                <LoadingButton
+                <LinkButton
                   className={classes.cta}
-                  href="mailto:contact@mobilic.beta.gouv.fr"
+                  to="mailto:contact@mobilic.beta.gouv.fr"
```

Comment:
:orange_circle: Accessibilité : il faut utiliser un lien et non un bouton pour mailto. LinkButton avec `to` créé un bouton (`Button`) et non un link (`MaterialLink`) (contrairement à lorsqu'il est utilisé avec `href`).
Plus d'infos : https://marijkeluttekes.dev/blog/articles/2024/11/04/html-link-or-button-that-is-the-question/

Reply:
Le Button DSFR crée bien un Link lorsqu'on renseigne un to dedans